copier/main.py

```diff
@@ -558,6 +558,13 @@ def _render_path(self, relpath: Path) -> Optional[Path]:
             if not part:
                 return None
             rendered_parts.append(part)
+
+        # The following logic was added based on the discussion: https://github.com/copier-org/copier/discussions/859
```

Comment:
This is not really needed, `git blame` will give you that!

If you want, you can add that kind of comment in the commit message though.

Reply:
Ok, I will remove this!

Reply:
Done!